src/csharp/Microsoft.Spark/Sql/Column.cs

```diff
@@ -589,7 +589,25 @@ public Column As(string alias)
         /// <returns>Column object</returns>
         public Column As(IEnumerable<string> alias)
         {
-            return Apply("as", alias);
+            return ApplyMethod("as", alias);
+        }
+
+        /// <summary>
+        /// Extracts a value or values from a complex type.
+        ///The following types of extraction are supported:
+        ///
+        ///1. Given an Array, an integer ordinal can be used to retrieve a single value.
+        ///2. Given a Map, a key of the correct type can be used to retrieve an individual value.
+        ///3. Given a Struct, a string fieldName can be used to extract that field.
+        ///4. Given an Array of Structs, a string fieldName can be used to extract field
+        ///of every struct in that array, and return an Array of fields.
+        ///
+        /// </summary>
+        /// <param name="Obj">object to apply</param>
+        /// <returns>Column object</returns>
+        public Column Apply(object Obj)
```

Comment:
Is there anything more meaningful than `Obj` ?  Same comment regarding the param description

Reply:
I'll change the paramname and description to something more meaningful